specs/dotsigstore_bundle/examples/blob_cert_rekor_timestamps.json

```diff
@@ -0,0 +1,38 @@
+{
+  "mediaType": "application/vnd.dev.sigstore.bundle+json;version=0.1",
+  "timestampVerificationData": {
+    "tlogEntries": [
+      {
+        "logIndex": "123",
+        "logId": "bG9naWQtMTIz",
+        "kindVersion": {
+          "kind": "hashedrecord",
+          "version": "v0.0.1"
+        },
+        "integratedTime": "1661340393",
+        "inclusionPromise": "U0VU"
+      }
+    ],
+    "rfc3161Timestamps": [
+      {
+        "signedTimestamp": "IyQlJg=="
+      }
+    ]
+  },
+  "verificationMaterial": {
+    "x509CertificateChain": {
+      "certificates": [
+        "MIIDKzCC",
+        "MIIEKzCC",
+        "MIIFKzCC"
+      ]
+    }
+  },
+  "messageSignature": {
```

Comment:
Pretty late to the party here..! but converting the PB definitions for the bundle to TypeScript generates a different looking object: https://github.com/sigstore/sigstore-js/pull/101/files?diff=unified&w=1#diff-3ae2d9489afcf6584e63bd44be81818ace67411f11346c301ab3ad16bff52bcaR40-R42 ([file link](https://github.com/sigstore/sigstore-js/blob/358452e8afc8e71e36e0a121d43e72aa1f738649/src/types/bundle/index.ts#L40-L42) if the pr link doesn't work)

Not sure if there's an easy fix to this but this disconnect from what's in code and what's in the json bundle is a bit confusing. The actual mapping happens in the generated typescript file: https://github.com/sigstore/sigstore-js/blob/358452e8afc8e71e36e0a121d43e72aa1f738649/src/types/bundle/__generated__/sigstore_bundle.ts#L83-L116

Is the type generator telling is to use generic/stable keys here instead? e.g.

```
...
"type": "messageSignature",
"data": {
  messageDigest": {
      "algorithm": "SHA2_256",
      "digest": "aGFzaA=="
    },
    "signature": "aGVq"
  }
}
```

Same applies to `verificationMaterial`, e.g.

```
...
"verificationMaterial": {
  "type": "x509CertificateChain",
  "data": {
    "certificates": [
      "MIIDKzCC",
      "MIIEKzCC",
      "MIIFKzCC"
    ]
  }
},
```

Reply:
I don't think this is an issue, it's an effect of how protobuf works, and the proto-json serializer/deserializer takes care this for the you. So how you actually access the content of an object might differ from how the "object looks" when serialized to JSON.

Reply:
👍  this makes sense. I'm still recovering from only working with json and json-schemas in JS land so found this initial disconnect a bit jarring.